.expeditor/config.yml

```diff
@@ -44,15 +50,41 @@ pipelines:
       description: Deploy the website
       definition: .expeditor/website.pipeline.yml
       trigger: pull_request
-  - habitat/build
   - release_habitat:
       description: Habitat release process
   - end_to_end:
       description: Habitat long running end to end tests - always runs on DEV environment
 
+staging_areas:
+  - post_merge:
+      workload: pull_request_merged:{{agent_id}}:*
+
 subscriptions:
   # These actions are taken, in order they are specified
+  - workload: pull_request_merged:{{agent_id}}:*
+    actions:
+      - trigger_pipeline:release_habitat
+
   - workload: buildkite_build_passed:{{agent_id}}:release_habitat:*
     actions:
       - trigger_pipeline:end_to_end:
           post_commit: true
+
+  # Make sure we unlock to allow later builds if one fails.
+  - workload: buildkite_build_failed:{{agent_id}}:release_habitat:*
+    actions:
+      - unlock_staging_area:post_merge:
+          post_commit: true
+          always_run: true
+
+  - workload: buildkite_build_passed:{{agent_id}}:release_habitat:*
+    actions:
+      - unlock_staging_area:post_merge:
+          post_commit: true
+          always_run: true
+
+  - workload: buildkite_build_cancelled:{{agent_id}}:release_habitat:*
```

Comment:
In the buildkite world, there is only one `l` in `canceled` (https://buildkite.com/docs/apis/rest-api/builds)

Reply:
Fixed :D